{"name": "反向用例: raise TypeError", "status": "passed", "attachments": [{"name": "log", "source": "f30dddad-ce08-4bf4-bb27-db31dad2e5e3-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "a", "value": "1"}, {"name": "b", "value": "[1, 2]"}, {"name": "error_type", "value": "'TypeError'"}], "start": 1624194207760, "stop": 1624194207760, "uuid": "a0fa3b11-4823-4bf9-aaa3-7adc260c3408", "historyId": "15db6cdeff522c9029531ceea01c555d", "testCaseId": "1b5aab2181c21923f6bd8223ee598e23", "fullName": "Test_2021_06_17.test_cal.test_cal.TestAddDivision#test_add_fail", "labels": [{"name": "story", "value": "测试加法"}, {"name": "feature", "value": "测试计算器"}, {"name": "parentSuite", "value": "Test_2021_06_17.test_cal"}, {"name": "suite", "value": "test_cal"}, {"name": "subSuite", "value": "TestAddDivision"}, {"name": "host", "value": "192.168.8.102"}, {"name": "thread", "value": "56187-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test_2021_06_17.test_cal.test_cal"}]}